cartography/intel/trivy/__init__.py

```diff
@@ -124,29 +126,73 @@ def sync_trivy_aws_ecr_from_s3(
     cleanup(neo4j_session, common_job_parameters)
 
 
+@timeit
+def sync_trivy_aws_ecr_from_dir(
+    neo4j_session: Session,
+    results_dir: str,
+    update_tag: int,
+    common_job_parameters: dict[str, Any],
+) -> None:
+    """Sync Trivy scan results from local files for AWS ECR images."""
+    logger.info(f"Using Trivy scan results from {results_dir}")
+
+    images_in_graph: set[str] = get_scan_targets(neo4j_session)
```

Comment:
Nit: clearer to name these as `image_uris`

Reply:
done